libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -205,9 +205,17 @@ void AP_Baro::calibrate(bool save)
     }
 
     if (hal.util->was_watchdog_reset()) {
-        BARO_SEND_TEXT(MAV_SEVERITY_INFO, "Baro: skipping calibration");
+        BARO_SEND_TEXT(MAV_SEVERITY_INFO, "Baro: skipping calibration after WDG reset");
         return;
     }
+    
+    #ifdef HAL_BARO_ALLOW_INIT_NO_BARO
+        if (_num_drivers == 0 || _num_sensors == 0 || drivers[0] == nullptr) {
```

Comment:
Formatting

Reply:
the brace's indentation?

Reply:
The "if" should line up with the braces above and below

Reply:
done